src/Documentation/Samples-Overview/Hello-World.md

```diff
@@ -5,6 +5,21 @@ title: Hello World
 
 # Hello World
 
+## Run the Hello world Sample
+One way run this sample is to download a local copy of HelloWorld from the 2.0 folder of the Orleans project.
```

Comment:
Did you mean "One way *to* run this sample"?

Reply:
Also, I'm afraid "Orleans project" may sound too ambiguous. Maybe simply "from https://github.com/dotnet/orleans/tree/master/Samples/2.0/HelloWorld/" or "from Samples/2.0/HelloWorld/ folder" instead?